skywalking-collector/skywalking-collector-cluster/src/main/java/com/a/eye/skywalking/collector/actor/AbstractClusterWorker.java

```diff
@@ -11,18 +11,45 @@
 import org.apache.logging.log4j.Logger;
 
 /**
+ * The <code>AbstractClusterWorker</code> should be implemented by any class whose instances
+ * are intended to provide receive remote message that message will transfer across different jvm
```

Comment:
The <code>AbstractClusterWorker</code> implementations represent workers, which receive remote messages.